crawler/src/main/java/com/movie/recommender/crawler/dataimport/MovieDataInitializer.java

```diff
@@ -12,17 +14,15 @@ public class MovieDataInitializer {
     private final MovieDataLoader movieDataLoader;
     private final MongoDBService mongoDBService;
 
+
     public MovieDataInitializer(MovieDataLoader movieDataLoader, MongoDBService mongoDBService) {
         this.movieDataLoader = movieDataLoader;
         this.mongoDBService = mongoDBService;
     }
 
     public void initializeData(String filePath) {
```

Comment:
add a condition to check if data exists (using `isDataInitialized` from `mongoDBService`)
if data exists, log a message and skip loading and insertion
otherwise, proceed with parsing and saving the new data